OCABundles/schema/bcgov-digital-trust/person/OCABundle.json

```diff
@@ -14,16 +14,9 @@
         "street_address": "Text"
       },
       "classification": "",
-      "digest": "ES3khN7HC1zZyr-DeQ6UbUVyhY_npOrxX_8NO_K4IWH8",
+      "digest": "EP4jXq7sdsWZUXFOdnmI2S9cKyw2SPKvuQdI_hL2lKGE",
       "flagged_attributes": [
-        "given_names",
```

Comment:
Confirming with Marcos that these attributes are not PII.  Since like at least the names and street address should be...

Reply:
@swcurran we have deflagged -- is that a word? -- most attributes in Person credential, and the new set (photo, DOB) is consistent with https://github.com/bcgov/bc-vcpedia/blob/main/guides/credential-ux-guide.md#flagged-attributes which states that "This overlay is intended to be used sparingly and is not meant for all personally identifiable information." That section of the UX Guide was reviewed and okayed by Oliver.